docs/pull-request-template.md

```diff
@@ -0,0 +1,21 @@
+## OPENEUROPA-[Insert ticket number here]
+
+### Description
+
+[Insert description here]
+
+### Change log
+
+- Added:
+- Changed:
+- Deprecated:
+- Removed:
+- Fixed:
+- Security:
+
+### Commands
```

Comment:
Can commands be a bit more specific? Are they installation commands? Debugging commands to test?

Reply:
The PR I added is intended to be an example and I took it directly from https://github.com/openeuropa/drupal-module-template/blob/master/github/PULL_REQUEST_TEMPLATE.md. In that context it refers to any commands that need to be executed after upgrading the component.

Reply:
Looking to previous closed PRs in next europa, the commands section that refers to commands that are required to enable/upgrade the component. Maybe the description in the example can be tweaked to make that clear.

Reply:
I would not include a Commands section here.
Commands should be automated and provided via code.
See NEPT-1628 for background info.

Reply:
Did not have knowdge about the ticket but definitely sounds better: commands that will need to be executed to make the PR work should be handled by code that can be run automatically in any environments and are not subjected to manual process/human error.

Reply:
I'll remove it then 👍